test/ui/data-models/common/DataModelListActionsMenu.test.js

```diff
@@ -4,10 +4,44 @@ import 'test/enzyme-init';
 import { shallow } from 'enzyme';
 import DataModelListActionsMenu from 'ui/data-models/common/DataModelListActionsMenu';
 
+const props = {
+  onClickDelete: jest.fn(),
+  payload: ({
+    modelId: 'ID',
+    descr: 'data model',
+    type: 'AAA',
+    model: '',
+    stylesheet: '',
+  }),
+};
+
 describe('WidgetListMenuAction', () => {
+  let component;
   beforeEach(jest.clearAllMocks);
   it('renders without crashing', () => {
-    const component = shallow(<DataModelListActionsMenu />);
+    component = shallow(<DataModelListActionsMenu />);
     expect(component.exists()).toBe(true);
   });
+
+
+  describe('test list component actions', () => {
+    beforeEach(() => {
+      component = shallow(<DataModelListActionsMenu {...props} />);
+      console.log(component.debug());
+    });
+
+    describe('with a row', () => {
+      it('has delete action', () => {
+        const dropdownKebab = component.find('DropdownKebab');
+        expect(dropdownKebab).toHaveLength(1);
+        expect(dropdownKebab.find('.DataModelMenuAction__delete')).toHaveLength(1);
+      });
+      it('verify click delete', () => {
```

Comment:
it's not necessary test the DropDownKebab action, because it'a tested by patternfly react library.